src/Ssh/Ssh/Common/RelayInformationUtils.cs

```diff
@@ -122,8 +128,15 @@ private bool CreateDefaultEndpoint(string id, out string exceptionMessage)
             }
             catch (ErrorResponseException exception)
             {
-                exceptionMessage = $"Unable to create default endpoint for the target Arc Server with error: {exception}. " +
-                    $"Contact Owner/Contributor of the resource.";
+                if (exception.Body.Error.Code == "AuthorizationFailed")
```

Comment:
That's a very good point. I don't believe it is ever localized, and if it is then it will still display the exception message in the else block. But that is something worth checking.

Reply:
Since it is an error code property I presume it would never be localized.  Just wanted to check.